### PR TITLE
Theme/topic: minor fixes

### PR DESCRIPTION
**Pull request by GormFrank**
**Including:**

- Background creating horizontal scroll on medium viewport;
- Removed unnecessary classes;
- Added inline fieldflow example;
- changed LinkedIn to true SVG.

### CODE REVIEW

File: site/pages/theme-topic-en.hbs [hunk 1]
```diff
@@ -52,6 +52,14 @@
 			<div class="col-md-4">
 				<h3><a href="#">[Subtopic hyperlink text]</a></h3>
 				<p>Summary of the information or tasks that can be accomplished on the sub-topic page. Remove prose or promotional messaging. Use action verbs.</p>
```
Comment (duboisp) [inline site/pages/theme-topic-en.hbs:54]: Please change the templated title in a way it suggested an action. At least for the description, like: Summary. Select a sub-category in the following drop down

**Review by duboisp — CHANGES_REQUESTED** [review]

**Comment by duboisp** [comment]
Other than the small description change, it's look all good.

**Review by duboisp — APPROVED** [review]

File: site/pages/theme-topic-en.hbs [hunk 2]
```diff
@@ -28,11 +28,11 @@
 <section class="well well-sm brdr-0 mrgn-tp-0">
 	<div class="container">
 		<div class="row">
-			<div class="pddng-r-0 col-md-2">
+			<div class="col-md-2">
 				<h2 class="h3 mrgn-tp-md">Most requested</h2>
```
Comment (GormFrank) [inline site/pages/theme-topic-en.hbs:32]:
```suggestion
				<h2 class="h4 mrgn-tp-md">Most requested</h2>
```

File: site/pages/theme-topic-en.hbs [hunk 3]
```diff
@@ -48,10 +48,18 @@
 <div class="container">
 	<section class="gc-srvinfo">
 		<h2>Services and information</h2>
```
Comment (GormFrank) [inline site/pages/theme-topic-en.hbs:50]:
```suggestion
		<h2>Services and information</h2>
```
```suggestion
		<h2 class="wb-inv">Services and information</h2>
```

File: site/pages/theme-topic-en.hbs (hunk 2 above)
Comment (GormFrank) [reply site/pages/theme-topic-en.hbs:32]: @delisma

File: site/pages/theme-topic-en.hbs (hunk 3 above)
Comment (GormFrank) [reply site/pages/theme-topic-en.hbs:50]: @delisma

**Pull request merged**